docs/conceptual/using-gpu-sanitizer.md

```diff
@@ -235,7 +235,7 @@ $ rocgdb <path to application>
 
 ### Using ASan with a short HIP application
 
-Refer to the following example to use address sanitizer with a short HIP application,
+Refer to the following example to use ASan with a short HIP application,
```

Comment:
@LisaDelaney - ASAN (all uppercase?)

Reply:
Looks good, Lisa. 

Did we put the abbreviation in parenthesis in the first instance? For example, AddressSanitizer (ASan). 
I noticed that LLVM recommends camel case, so we should retain the guideline for consistency. 

Thanks for your response.

Reply:
@Rmalavally - Yep, it's also in parentheses in the title, which I have mixed feelings about. This convention is not consistent across the docs, so probably warrants a conversation at some point as to whether we want to do this or not.

Reply:
> @Rmalavally - Yep, it's also in parentheses in the title, which I have mixed feelings about. This convention is not consistent across the docs, so probably warrants a conversation at some point as to whether we want to do this or not.

For sure. Let's discuss.